### DISCUSSION TITLE
Prisma raw query and PSQL give different results.

### DISCUSSION THREAD

**Discussion by fkaanoz** [prisma/prisma#14772] [Q&A] [answered]
Hi,

```
  const query = `SELECT * FROM "Users" WHERE name ILIKE '%${name.trim()}%' OR surname ILIKE '%${surname.trim()}%'`;
      console.log(query);

      const results =
        await prisma.$queryRaw`SELECT * FROM "Users" WHERE name ILIKE '%${name.trim()}%' OR surname ILIKE '%${surname.trim()}%'`;
      console.log(results);
```

This my code snippet is from one of my api endpoint. 

When I request,
Query was outputting as :  ` SELECT * FROM "Users" WHERE name ILIKE '%ds%' OR surname ILIKE '%ad%' ` 
Results was outputting as  empty array. 
However, when I ran this query in psql, I am getting a result as I expected. 

Where is the mistake? 
Thank you.

Note : I am totally copying and pasting the query.  

Console Screenshot : 
![image](https://user-images.githubusercontent.com/61204681/184218212-a63aa1f5-1ce6-4311-832b-73d94ff30639.png)

psql Screenshot : 
![image](https://user-images.githubusercontent.com/61204681/184218394-877a6bb5-684f-4461-8861-20c7461d6349.png)

**Comment by ludralph** [comment, marked as answer]
Hi @fkaanoz 👋 

I was able to reproduce your issue. You would need to tweak your queryRaw statement like the code below

```js
const name = 'ds'
const surname = 'ada'
const results =
    await prisma.$queryRaw`SELECT * FROM "products" WHERE name ILIKE ${`%${name.trim()}%`} OR surname ILIKE ${`%${surname.trim()}%`}`;
    console.log(results);
    prisma.$on('query', (e) => {
      console.log('Query: ' + e.query)
    })
```
The output is
```
prisma:query SELECT * FROM "products" WHERE name ILIKE $1 OR surname ILIKE $2
[ { id: 1, name: 'adsas', surname: 'dadads' } ]
```

**Reply by fkaanoz** [reply]
thank you so much.

**Reply by ludralph** [reply]
You're welcome.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
